src/metrics/metrics.jl

```diff
@@ -356,10 +354,14 @@ function _shelter_species_loop(
     # ASV should be 0.0 where MSV is 0.0 so the end result is 0.0 / 1.0
     MSV[MSV.==0.0] .= 1.0
 
+    # Number of functional groups
+    n_groups::Int = size(MSV, 1)
+    # Number of size classes
+    n_sizes::Int = Int(n_species / n_groups)
     # Loop over each taxa group
 
-    RSV::YAXArray = ZeroDataCube((:timesteps, :species, :sites), size(X[species=1:6]))
-    taxa_max_map = zip([i:i+5 for i in 1:6:n_species], 1:6)  # map maximum SV for each group
+    RSV::YAXArray = ZeroDataCube((:timesteps, :species, :sites), size(X[species=1:n_groups]))
+    taxa_max_map = zip([i:(i + n_sizes - 1) for i in 1:n_sizes:n_species], 1:n_groups)  # map maximum SV for each group
```

Comment:
`n_species` -> `n_group_sizes` 

I know it's a horrible name too, but at least if we standardize it, we can do a search and replace later.